platform/ext/target/nordic_nrf/nrf9160/partition/region_defs.h

```diff
@@ -104,7 +104,10 @@
 #define S_CODE_LIMIT    (S_CODE_START + S_CODE_SIZE - 1)
 
 #define S_DATA_START    (S_RAM_ALIAS(0x0))
-#define S_DATA_SIZE     ((TOTAL_RAM_SIZE / 2) - 0x4000) /* 112 KB */
+/* Assign to SPE the minimum amount of RAM (aligned to the SPU region boundary)
+ * that is needed for the most demanding configuration, which turns out to be
+ * the RegressionIPC one. */
```

Comment:
Did you test this on the PSA tests?

Reply:
Yes, I tried it with storage tests, and those needed ~62 kB (in the IPC mode, ~48 kB otherwise) of RAM for PSE.